src/centralerekenapplicatie_v1/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from django.utils import timezone
+
+from contrib.timescale.fields import TimescaleDateTimeField
+
+
+class CraModel(models.Model):  # TODO: We need another name for this model
+    created_at = models.DateTimeField(auto_now_add=True)
+    message_id = models.TextField()                     # Coming from "id" in the message.
+    type = models.CharField(max_length=255)             # Either areaMetrics or lineMetrics
+    sensor = models.CharField(max_length=255)  # e.g. "CMSA-GAWW-17"
+    timestamp = models.DateTimeField()  # TODO Convert to timescale field
```

Comment:
Omdat dit een nieuwe app is kunnen we het beste timescale direct inregelen toch? Dan hoeft er ook geen datamigratie aan te pas te komen.